.github/workflows/development.yml

```diff
@@ -0,0 +1,47 @@
+name: Test Builds On Merge Creation
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+  testbuild:
+    name: Test Compile On Pull Request
+    runs-on: ubuntu-22.04
+    env:
+      OUTPUT_PATH: output/get5
+      OUTPUT_SM_PATH: output/get5/addons/sourcemod
+      SM_JSON_INC_PATH: get5/dependencies/sm-json/addons/sourcemod/scripting/include
+      STEAMWORKS_URL: https://raw.githubusercontent.com/KyleSanderson/SteamWorks/1.2.3c/Pawn/includes/SteamWorks.inc
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: 'get5'
+        submodules: true
+
+    - uses: rumblefrog/setup-sp@master
+      with:
+        version: '1.10.x'
+
+    - name: Get shorthand commit.
+      id: vars
+      run: |
+        echo "::set-output name=sha_short::$(echo ${{ github.sha }} | cut -c1-6)"
+        echo "::set-output name=get5_version::$(cat get5/scripting/get5/version.sp | grep -Eo \"[0-9]*\.[0-9]\.[0-9]-[a-zA-z]{3}\")"
+
+    - run: | 
+        wget $STEAMWORKS_URL -P steamworks
+        mkdir -p $OUTPUT_SM_PATH/plugins/disabled
+        sed -i -r "s#[0-9]*\.[0-9]*\.[0-9]*-[a-zA-z]{3}#${{ steps.vars.outputs.get5_version }}-${{ steps.vars.outputs.sha_short }}#g" get5/scripting/get5/version.sp
```

Comment:
Same here. Better, but still allows something like `.3.3-dev`. Intentional?

Reply:
Intentional as we should be sticking to semantic versioning. If there's an error in the version number, it will at least be shown immediately on tag creation? Hopefully something that would be caught way sooner. Also fixed this one up in build.yml as well.

Reply:
I don't understand. You want it to actually match to an invalid syntax and not error?

Reply:
Actually wait, with the new regex match it will still match something like `.12.2-dev`, so it will show up on tag creation without error. I feel like this is something that should not stop build creation though, as it's only a versioning number?